browser/brave_browser_main_parts.cc

```diff
@@ -51,7 +51,7 @@
 #include "components/sync/driver/sync_user_settings.h"
 #endif
 
-#if BUILDFLAG(BRAVE_WALLET_ENABLED) && BUILDFLAG(ENABLE_EXTENSIONS)
+#if BUILDFLAG(ETHEREUM_REMOTE_CLIENT_ENABLED) && BUILDFLAG(ENABLE_EXTENSIONS)
```

Comment:
is there a brave_wallet buildflag dep/header that we can remove now?

Reply:
There wasn't one to begin with, I think it had it before through a diff header but thought it better to include it here.